### ISSUE TITLE
Write text description of Gonad sample stages

### ISSUE THREAD

**Issue by sr320** [RobertsLab/project-geoduck-repro#1]
While the classification is more detailed for your original work - please come up with some accurate text that would describe the three stages used in the proteomics manuscript..

for reference here is the draft manuscript.

you can add text on google doc or in comment below - thanks

**Comment by grace-ac** [comment]
Females
Early: Either no secondary oocytes or secondary oocytes that are ~5-15µ
Mid: Secondary oocytes ~50-70µ
Late: Secondary oocytes ~65-85µ

Males
Early: Mostly somatic cells and ~5% spermatids per acinus
Mid: Medium somatic cell coverage and ~50% spermatids per acinus
Late: Very little somatic cell coverage and ~75-90% spermatids per acinus

**Comment by sr320** [comment]
Nice! how about  in paragraph form...?

**Comment by grace-ac** [comment]
Female reproductive maturation staging is characterized by early stage maturity having either no secondary oocytes, or oocytes that measure ~5-15µ. Middle stage maturity is characterized as having oocytes measuring ~50-70µ. Late stage maturity is characterized as having oocytes measuring ~65-86µ. Male reproductive maturation staging is characterized by early stage maturity having mostly somatic cells and ~5% spermatid composition per acinus. Middle stage maturity is characterized as having about equal composition of somatic cells and reproductive tissue and ~50% spermatid composition per acinus. Late stage maturity is characterized as having very little somatic cells and ~75-90% spermatid composition per acinus.

**Comment by grace-ac** [comment]
I may have been too wordy...

**Comment by sr320** [comment]
One more time with less words...

something like 
early stage (no secondary oocytes, or oocytes that measure ~5-15µ), middle stage (....

**Comment by grace-ac** [comment]
Female reproductive maturation stages were categorized as early (no secondary oocytes, or oocytes that measure ~5-15µ), middle (secondary oocytes ~50-70µ), and late (secondary oocytes ~65-85µ). Male reproductive maturation stages were characterized as early (mostly somatic cells and ~5% spermatid composition per acinus), middle (about equal parts somatic cells and reproductive tissue and ~50% spermatid composition per acinus), and late (very little somatic cells and ~75-90% spermatid composition per acinus).

**Comment by grace-ac** [comment]
Ah... that actually looks very similar in length to my previous attempt. I'll try again.

**Comment by sr320** [comment]
No need - looks great! - And it is accurate based on the samples that were used in the paper?

**Comment by grace-ac** [comment]
oh yay!! and yes- i double checked the samples with the stages i have in my capstone paper.
